backend/dataall/api/Objects/Environment/input_types.py

```diff
@@ -28,7 +28,6 @@
         gql.Argument('description', gql.String),
         gql.Argument('AwsAccountId', gql.NonNullableType(gql.String)),
         gql.Argument('region', gql.NonNullableType(gql.String)),
-        gql.Argument('dashboardsEnabled', type=gql.Boolean),
```

Comment:
there is 1 more usage in the resolvers which I think was missed: https://github.com/awslabs/aws-dataall/blob/modularization-main/backend/dataall/api/Objects/Environment/resolvers.py#L130C7-L130C7

Reply:
I missed it intentionally, since it was a request parameter. But now, I think it's better to migrate it as well :)